buildinfo_test.go

```diff
@@ -223,6 +223,8 @@ func TestArtifactoryPublishBuildInfoBuildUrl(t *testing.T) {
 	initArtifactoryTest(t)
```

Comment:
This test is pretty good. I suggest we also test that if a build name and build number are set by env vars (and not by command options), the env vars are used.

Reply:
This is already tested and in 2 levels:
Unit test: TestExtractBuildDetailsFromEnv
Integration test: TestArtifactoryGenericBuildnameAndNumberFromEnv